tests/test_kernel_language.py

```diff
@@ -265,7 +265,7 @@ def kernel(particle, fieldset, time):
 
 
 def random_series(npart, rngfunc, rngargs, mode):
-    random = parcels_random if mode == 'jit' else py_random
+    random = ParcelsRandom if mode == 'jit' else py_random
     random.seed(1234)
     func = getattr(random, rngfunc)
     series = [func(*rngargs) for _ in range(npart)]
```

Comment:
yes, exactly - these parts are the ones that become quite difficult of all the different libraries are named 'random'.